components/molecules/client/user/user-small.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+
+interface UserSmallProps {
+  avatarUrl: string;
+  name: string;
+}
+
+const UserSmallComponent: React.FC<UserSmallProps> = ({ avatarUrl, name }) => {
+  return (
+    <div className="flex items-center w-full h-full gap-sm">
+      {/* Image Box with Ellipse Shape */}
+      <div className="w-fit h-fit rounded-full overflow-hidden flex-shrink-0">
+        <Image
+          src={avatarUrl}
+          alt={name}
+          width={25}
+          height={25}
+          className="object-cover w-full h-full rounded-lg"
```

Comment:
Use w-[25px] h-[25px]

Reply:
replaced w-full h-full with w-[25px] h-[25px].

Reply:
also removed this for redundancy:
width={25}
          height={25}